### PR TITLE
Add Onboarding flow (CU-hjwagg)

### PR DESCRIPTION
**Pull request by trisapeace**
- For now, I've just used some of the icons as the images in the Onboarding flow. I think we could probably really get away with those, but I'll still clear it with Kenzie as part of https://app.clickup.com/t/hjwaec .

Some screenshots: 
image
image
image
simulator_screenshot_7383FD52-89AD-4D1C-81E2-47ADC6C9FB48

### CODE REVIEW

File: src/components/Cards/ContactBraveBoxesCard.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import colors from '../../resources/colors'
+
+function ContactBraveBoxesCard() {
+  return (
+    <View style={styles.container}>
+      <Text>Still have questions? We have answers.</Text>
+
+      {/* TODO add InfoBox with text "Learn more" and a link to https://bravesensor.com/alertfaq */}
+
+      {/* TODO add ContactBraveBoxes here once PR #4 is merged */}
```
Comment (trisapeace) [inline src/components/Cards/ContactBraveBoxesCard.jsx:15]: Reminder to add the "Learn More" info back and ContactBraveBoxes once #4 is merged.

File: src/components/Cards/ContactBraveBoxesCard.jsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  )
+}
+
+const styles = StyleSheet.create({
+  container: {
+    alignItems: 'center',
+  },
+  text: {
+    color: colors.greyscaleDark,
+    // fontFamily: 'Roboto' // TODO after mering #4
```
Comment (trisapeace) [inline src/components/Cards/ContactBraveBoxesCard.jsx:26]: Reminder to add the `fontFamily` to all the `Text` s after #4 is merged.

File: src/navigation/RootStack.jsx [hunk 3]
```diff
@@ -5,16 +5,29 @@ import { createStackNavigator } from '@react-navigation/stack'
...
 import MainTabs from './MainTabs'
+import OnboardingScreen from '../screens/OnboardingScreen'
 import ErrorScreen from '../components/ErrorScreen'
 import SCREEN from './ScreensEnum'
 
 const Stack = createStackNavigator()
 
 function RootStack() {
+  // TODO After merging PR #6, change the initialRouteName to SCREEN.MAIN if a location has been set in the store
+  const initialRouteName = SCREEN.ONBOARDING
```
Comment (trisapeace) [inline src/navigation/RootStack.jsx:16]: Reminder to do this after merging #6

**Comment by trisapeace** [comment]
Task linked: [CU-hjwagg Frontend & Backend: Onboarding flow: Implementation](https://app.clickup.com/t/hjwagg)

File: src/components/Cards/ContactBraveBoxesCard.jsx (hunk 1 above)
Comment (trisapeace) [reply src/components/Cards/ContactBraveBoxesCard.jsx]: Done

File: src/components/Cards/ContactBraveBoxesCard.jsx (hunk 2 above)
Comment (trisapeace) [reply src/components/Cards/ContactBraveBoxesCard.jsx]: Done.

File: src/navigation/RootStack.jsx (hunk 3 above)
Comment (trisapeace) [reply src/navigation/RootStack.jsx]: Done.

**Comment by trisapeace** [comment]
@schwarrrtz I'm going to merge this but still leave the Task in Blocked so that you can review when you get the chance.

**Comment by schwarrrtz** [comment]
I've finally gotten React Native fully functional on my new computer and ran this on the Android emulator. It looks great overall. The only two things that jump out at me right now are:

- the body text on the onboarding cards seems too small and light to be easily readable
- Storybook still seems to be a bit messed up; various components just show up as black vertical lines, and the InfoBox component looks pretty different from what it used to look like. This may be an issue with my machine and also the Storybook weirdness was happening before you merged this @trisapeace so we can definitely address that separately.

**Pull request merged**